entity-framework/core/miscellaneous/cli/dotnet.md

```diff
@@ -74,9 +74,9 @@ Before using the tools, you might have to create a startup project or set the en
 
 The commands refer to a *project* and a *startup project*.
 
-* The *project* is also known as the *target project* because it's where the commands add or remove files. By default, the project in the current directory is the target project. You can specify a different project as target project by using the <nobr>`--project`</nobr> option.
+* The *project* is also known as the *target project* because it's where the commands add or remove files. By default, the project in the current directory is the target project. You can specify a different project as target project by using the `--project` option.
 
-* The *startup project* is the one that the tools build and run. The tools have to execute application code at design time to get information about the project, such as the database connection string and the configuration of the model. By default, the project in the current directory is the startup project. You can specify a different project as startup project by using the <nobr>`--startup-project`</nobr> option.
+* The *startup project* is the one that the tools build and run. The tools have to execute application code at design time to get information about the project, such as the database connection string and the configuration of the model. By default, the project in the current directory is the startup project. You can specify a different project as startup project by using the `--startup-project` option.
```

Comment:
https://github.com/dotnet/EntityFramework.Docs/pull/2780/files

Reply:
I'm not seeing it a value looking at the live site for `white-space: nowrap` on the in-line code sections. I can remove it, but only some elements that are using it seem to be affected, and other elements in the table that are, don't have the elements.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr

Reply:
So should we be replacing `<nobr>` with `<span style="white-space: nowrap;">` rather than removing it altogether?

Reply:
I could just undo this part while you figure out if there is an upstream CSS issue that should be looked at

Reply:
I've opened #2782 to track this - any idea who'd know about this in the doc teams? If you want to undo that part so we can merge this PR, go ahead, or we can wait for that.

Reply:
I usually bug @mairaw when I run into infrastructure stuff on docs stuff 😆 
The main place I know of for tracking overall docs.microsoft.com stuff is https://github.com/microsoftdocs/feedback

For now, I'm just going to rebase out this commit so the rest can be reviewed